server/server.go

```diff
@@ -41,14 +41,18 @@ var (
 	errInvalidMaxAsyncTaskCount = errors.New("invalid max async task count")
 )
 
+type Handlers []Handler
+
 type Server struct {
-	Logger         raknet.Logger
-	Handler        Handler // TODO: supports multiple handlers
+	Logger         utils.Logger
+	Handlers        Handlers
 	MaxConnections int
 	MTU            int
-	Identifier     identifier.Identifier
+	Identifier     identifier.Minecraft
```

Comment:
go-raknet is just a raknet server library.
If it changes, goraknet will be dedicated for minecraft.

Reply:
Others is very good job!

Reply:
yeah i guess you're right

Reply:
Could you commit a fixed code to the PR?

Reply:
or do you want me to commit?